DQM/HcalTasks/plugins/HcalGPUComparisonTask.cc

```diff
@@ -115,7 +145,8 @@ HcalGPUComparisonTask::HcalGPUComparisonTask(edm::ParameterSet const& ps)
     if (mRecHitEnergy.find(did) == mRecHitEnergy.end())
       mRecHitEnergy.insert(std::make_pair(did, energy));
     else
-      edm::LogError("HcalDQM|RechitTask") << "Duplicate Rechit from the same HcalDetId";
+      edm::LogError("HcalGPUComparisonTask") << "Duplicate Rechit from the same HcalDetId" << std::endl;
+    ;
```

Comment:
`std::endl` here is not needed, as `Log*` calls ignore it and apply a newline regardless.